src/main/java/com/powsybl/openloadflow/ac/equations/HvdcAcEmulationSide1ActiveFlowEquationTerm.java

```diff
@@ -24,9 +24,13 @@ public HvdcAcEmulationSide1ActiveFlowEquationTerm(LfHvdc hvdc, LfBus bus1, LfBus
     }
 
     private double p1(double ph1, double ph2) {
-        double rawP = rawP(p0, k, ph1, ph2);
-        double boundedP = boundedP(rawP);
-        return (isController(rawP) ? 1 : getVscLossMultiplier()) * boundedP;
+        double rawP = boundedP(rawP(p0, k, ph1, ph2));
+        if (isController(rawP)) {
+            return rawP;
+        } else {
+            double rectifierPDc = (1 - lossFactor1) * rawP;
```

Comment:
Can you try to put more commun code in the abstract?

Reply:
Some modifications have been done, especially by creating a method getAbsActivePowerWithLosses() that computes the absolute value of the active power taking into account loss of converters (in the correct order depending on the direction of the transit) and the line loss.